assemblies/assembly_openshift-trial/main.adoc

```diff
@@ -0,0 +1,27 @@
+[id='assembly_openshift-authoring]
+
+include::_artifacts/document-attributes.adoc[]
+
+= Deploying a {PRODUCT} {PRODUCT_VERSION} trial environment on {OPENSHIFT}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: openshift-authoring
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a system engineer, you can deploy a {PRODUCT} trial environment on {OPENSHIFT} to evaluate or demonstrate development and use of processes and other business assets.
```

Comment:
"use of business assets" or "rules and other business assets", to make it apply to both DM and PAM, rather than ifdefing and rather than using your :ASSETS: attribute, which I suggested to nix in my previous comment.